quickstarts/microsoft.kubernetes/fleet/private-hubful-fleet-deploy.json

```diff
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
consider reusing camel case like you did above, and writing fleetName rather than fleets_priv_fleet_name ?